bmwqemu.pm

```diff
@@ -182,55 +185,43 @@ sub get_timestamp {
     return sprintf "%s.%04d ", (POSIX::strftime "%H:%M:%S", gmtime($t)), 10000 * ($t - int($t));
 }
 
-sub print_possibly_colored {
-    my ($text, $color) = @_;
-
-    if (($direct_output && !$istty) || !$direct_output) {
-        $logfd && print $logfd get_timestamp() . "$$ $text\n";
-    }
-    if ($istty || !$logfd) {
-        if ($color) {
-            print STDERR colored(get_timestamp() . "$$ " . $text, $color) . "\n";
```

Comment:
as you remove colored calls, you can remove the Term::ANSIColor dependency as well

Reply:
Done.

Removed from the imports and from the cpanfile

Reply:
@okurz Only for STDOUT/STDERR, correct?

If yes, I'm fine with it. Please open a poo.

Reply:
I did with https://progress.opensuse.org/issues/34978 , thx